### ISSUE TITLE
cap issue report on Coptic

### ISSUE THREAD

**Issue by GoogleCodeExporter** [googlei18n/noto-fonts#312]
```
Issues on character character coverage, design and layout are reported in the 
attached file.
```

Original issue reported on code.google.com by `xian...@google.com` on 27 Mar 2015 at 11:06

Attachments:
- [Coptic review.pdf](https://storage.googleapis.com/google-code-attachments/noto/issue-312/comment-0/Coptic review.pdf)

**Comment by behdad** [comment]
cc @xiangyexiao

**Comment by kmansourMT** [comment]
The Coptic review PDF is a large collection of wishes, along with some error reports. Some of the wishes include additional characters that were not in the original specs, and therefore cannot be considered bugs. We will very soon be supplying a new version of the font that fixes all the technical errors. We have also heeded some of the design "wishes" and have made changes to improve the product. Once the Google team has received the new version, we can address any remaining issues.

**Comment by kmansourMT** [comment]
Following is my initial reply (May 2015) to the review and the Lint report:
2015.4.27 km

Lint report:
	Missing characters u0374 & u375.
	These characters can be readily copied from Noto Sans LGC module.

Issue 312: report by John Hudson

Claim 1: Glyph set: why are some listed IPA characters included?
Reply: remove shown characters because they aren’t needed.

Claim 2: font does not contain the historic Coptic epact numbers, which have now been 
	included in Unicode (block 102E0)
Reply: this range of characters was not required when specification was agreed upon.
	
Claim 3: Design: various recommendations for improving design of a handful of characters
Reply: Some of the suggestion make sense and would constitute an improvement in terms
	of style and legibility. I would like to review these with Carl C. or another designer to finalize.

Claim 4: Layout behavior
	Kerning: font contains no kerning, but needs it for some combinations. (in particular for
	all-cap settings)
Reply: I’m not sure this was a requirement at the time or if it’s a nice afterthought. Let’s discuss
	to determine how much work would be involved in cap-to-cap kerning pairs.

Claim 5: Layout behavior
	Overline marks: the overline features (with various types of marks) appear to fail when the base
	characters are of a different height.
Reply: The coding of this feature was done in Volt; I believe it can be fixed with a modest amount
	of effort by using a new, clever technique.

Claim 6: Layout behavior
	Continuos macron: claim that it does not render correctly under any environment, including
	Windows 7. 
Reply: We need to verify the functionality of this feature under Volt, as well as under
	Windows 8.

**Comment by marekjez86** [comment]
per MT "Issues were addressed in v1.03 TTF 2015 delivery"
